homeassistant/components/nina/__init__.py

```diff
@@ -34,7 +34,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     regions: dict[str, str] = entry.data[CONF_REGIONS]
 
-    coordinator = NINADataUpdateCoordinator(hass, regions)
+    filter_corona: bool = entry.data[CONF_FILTER_CORONA]
+
+    coordinator = NINADataUpdateCoordinator(hass, regions, filter_corona)
```

Comment:
```suggestion
    coordinator = NINADataUpdateCoordinator(hass, regions, entry.data[CONF_FILTER_CORONA])
```
We don't need this intermediate variable